interp/constrintern.ml

```diff
@@ -2048,7 +2048,11 @@ let internalize globalenv env pattern_mode (_, ntnvars as lvar) c =
           assert (Option.is_empty isproj);
           let c = intern_notation intern env ntnvars loc ntn ntnargs in
           find_appl_head_data c, args
-        | _ -> assert (Option.is_empty isproj); (intern_no_implicit env f,[],[]), args in
+        | _ ->
+           assert (Option.is_empty isproj);
+           let f = intern_no_implicit env f in
+           let f, _, args_scopes = find_appl_head_data f in
+           (f,[],args_scopes), args in
```

Comment:
Please put the ending `in` of a multiline `in` on a separate line (I find it harder to understand the code structure otherwise).

Reply:
Ah, I always do like this... Could we then try to state an explicit recommendation about this point in [`dev/doc/styles.txt`](https://github.com/coq/coq/blob/33e748514dad9459885006a1523d107d556be22b/dev/doc/style.txt), that I can then follow consistently.

Reply:
@skyskimmer: I finally did as you suggested.

After more thinking, I believe that the change is natural enough to not need prior comments from users. So, looking for an @coq/extensible-syntax-maintainers's assignee (it is very short, so, it should not take more than 5 minutes).